quantum/dynamic_macro.h

```diff
@@ -278,10 +286,10 @@ bool process_record_dynamic_macro(uint16_t keycode, keyrecord_t *record)
             /* Store the key in the macro buffer and process it normally. */
```

Comment:
I would add `DYN_MACRO_PLAY1` and `DYN_MACRO_PLAY2` to this switch case, and just return false and log if they are pressed instead of adding the extra parameter and handling the failure in the function.